libs/openFrameworks/gl/ofGLProgrammableRenderer.cpp

```diff
@@ -1315,12 +1321,46 @@ void ofGLProgrammableRenderer::bind(const ofFbo & fbo, bool setupPerspective){
 }
```

Comment:
how does this start the fbo at all? fbo.begin() calls this but none bind the fbo, right?